### PR TITLE
sf4 compatibility

### PR DESCRIPTION
**Pull request by harikt**

### CODE REVIEW

**Comment by harikt** [comment]
The build failure is for 5.3, 5.4, 5.5 ( for dependency on silex raised to 2.x , silex itself is not maintained any more) . Is this still supporting those php versions ?

Screenshot from 2019-06-10 13-08-48

**Comment by peter279k** [comment]
I think it;s fine to drop `php-5.3`, `php-5.4` and `php-5.5` because they've been unsupported for about whiles.

**Comment by harikt** [comment]
Hey @simensen ,

Is there a way we can proceed here ?

**Comment by fabpot** [comment]
Let's drop unsupported PHP versions

File: .travis.yml [hunk 1]
```diff
@@ -6,12 +6,11 @@ cache:
...
-    - "5.4"
-    - "5.5"
-    - "5.6"
-    - "7.0"
-    - "hhvm"
+    - 7.1
+    - 7.2
+    - 7.3
+    - 7.4snapshot
+    - nightly
```
Comment (fabpot) [inline .travis.yml]: I would remove nightly.

Comment (harikt) [reply .travis.yml]: Yes, I removed it. Also removed 7.4snapshot for it was also causing issue downloading symfony/router from packagist. 
Screenshot from 2019-06-20 13-04-36

**Comment by harikt** [comment]
Finally resolved things.

File: .travis.yml (hunk 1 above)
Comment (peter279k) [reply .travis.yml]: If we want add the `php-nightly`, it should add this in [allow_failures ](https://docs.travis-ci.com/user/build-matrix/#matching-jobs-with-allow_failures)block.

Nobody can guarantee that the `php-nightly` will be successful in every Travis CI build.

Comment (harikt) [reply .travis.yml]: Seems that was wrong due to some reason and 7.4 is working fine.

Comment (harikt) [reply .travis.yml]: if we allow failure we probably don't care it. So may be it is not needed for now ?

**Comment by fabpot** [comment]
Thank you @harikt.

**Comment by fabpot** [comment]
1.2.0 released

**Pull request merged**